packages/pwa/app/translations/README.md

```diff
@@ -89,6 +89,21 @@ For example, in your React component, you can add formatted messages like `intl.
 
 At the minimum, only defaultMessage is the required parameter. The message id is optional. If you don’t specify it, the id is auto-generated for you.
 
+### Creating message id
+
+Message ids is optional. If it is omitted, the ids will be generated automatically when you extract and compiled default messages.
+However, if you find that it is best to create message id manually, it is advised to include three elements as below order to provide the best context for translators 
+
+- Location: the place where this message is located (e.g component/page name, “global”) 
+- UI Type.  
```

Comment:
@vmarta As per action from our last discussion, we do want to keep the UI type strict, but since this is open sourced, it does not make sense to introduce a list of type UI here, so I am open to discuss on what we want to write down here

Reply:
Hmm, good point about the open source aspect. I'm not sure at the moment 🤔

Reply:
It's not wise to tell people how to format their id's, everyone is going to do it a little different. I'd be weary about this entire section about message id's since it's used during out internal process of translating.

Reply:
That is a good point. I will take it out, and maybe should have our internal doc for the id format.